CrawlJob.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Simgroep\ConcurrentSpiderBundle;
+
+use PhpAmqpLib\Message\AMQPMessage;
+
+class CrawlJob
+{
+    private $url;
+    private $baseUrl;
+    private $blacklist = array();
```

Comment:
it is needed to set this to array if its done in constructor(array $blacklist = array()), so it always be array if nothing passed to constructor. And the same situation for $metadata